wiki/Beatmapping/Star_Rating/en.md

```diff
@@ -0,0 +1,19 @@
+---
+tags:
+    - SR
```

Comment:
```suggestion
  - SR
```
might as well stick to convention.

Reply:
aaa, any way i can make do toggle this on by default in my editor?

Reply:
ok i figured out by not being lazy!